pkgs/by-name/gv/gvmd/package.nix

```diff
@@ -0,0 +1,114 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, cmake
+, gvm-libs
+, pkg-config
+, gnutls
+, xml-tooling-c
+, glib
+, doxygen
+, git
+, libbsd
+, libical
+, libpkgconf
+, gpgme
+, mount
+, libidn2
+, graphviz
+, icu
+, p11-kit
+, pcre2
+, libxslt
+, libtasn1
+, postgresql
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  nativeBuildInputs = [
```

Comment:
Putting all the stuff into `nativeBuildInputs` is unwise and can break a lot of assumptions, not only cross compilation. Please add `strictDeps = true` to see which packages should go to `buildInputs`.